v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FExtractionsAndTransforms.scala

```diff
@@ -48,9 +46,16 @@ object V2FExtractionsAndTransforms {
   ): SCollection[(String, Msg)] = {
     msgAndFilePaths.map {
       case (path, msg) =>
+        // change to snake case
+        val withSnakeCase = MsgTransformations.keysToSnakeCase(msg)
```

Comment:
I'm pretty sure this is redundant at the point where we call `extractAndTransformVariants`

Reply:
I don't think it is; all that has been called on the data at this point is `extractAndConvert` which doesn't make the snake case adjustment. That doesn't happen till `transform` is called. I can move that into `extractAndConvert` to keep the code DRYer.

Reply:
Thinking about it more, let's save any big refactorings for a follow-up PR. The `**.csv` is the only one that feels good to sneak in.